src/Sdk/Cerbos/Sdk/Cloud/V1/Store/GetFilesRequest.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021-2025 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+
+namespace Cerbos.Sdk.Cloud.V1.Store
+{
+    public sealed class GetFilesRequest
+    {
+        private string StoreId { get; set; }
+        private List<string> Files { get; }
+
+        private GetFilesRequest()
+        {
+            Files = new List<string>();
+        }
+        
+        public static GetFilesRequest NewInstance()
```

Comment:
Again, I'd make mandatory arguments part of the constructor itself so that it leaves less room for users to make mistakes. 
```suggestion
        public static GetFilesRequest NewInstance(string storeId, string[] paths)
```